roombasim/pittras/ai/velocity_demo_controller.py

```diff
@@ -0,0 +1,23 @@
+'''
+velocity_demo_controller.py
+'''
+
+from roombasim.ai import Controller
+
+def velocity_task_completion_callback(status):
+    '''
+    Callback for Velocity task completion.
+    '''
+    print("Velocity task completed with", status)
+
+class VelocityDemoController(Controller):
+    '''
+    A demo controller tests VelocityTask.
+    '''
+
+    def setup(self):
+        self.task_controller.switch_task(
+            'VelocityTask',
+            callback=velocity_task_completion_callback,
```

Comment:
Probably doesn't matter for the sake of the demo controller, I am questioning whether the callback method should be a method in `VelocityDemoController` rather than outside the class.